### ISSUE TITLE
Resolve issues on internal names when default search domain is set in host OS

### ISSUE THREAD

**Issue by kvij** [lando/lando#1644]
**Setup**

Lando v3.0.0-rc.16 on Windows 10 Professional

**Relevant part of `.lando.yml`**

```yml
services:
  smtp:
    type: mailhog
    hogfrom:
      - appserver
```

**Tell us about the error you got**

Usualy a network timeout or other error/weird stuf that happens when one get the wrong IP on a DNS request.

**Tell us generally about your bug**

On Windows (not on OS X) the default search domain is added to the containers /etc/resolv.conf. 
For a lot of services this is not a problem but for `mhsendmail` it is. As internal service names are no FQDN the default search domain is appended.
This results in completely wrong resolves

**Steps to reproduce**

1. Have a default search path in Windows DNS
2. Use `mhsendmail` or any other golang app that uses `net` [name resolution](https://golang.org/pkg/net/#hdr-Name_Resolution) as (`net.Dial` does)
3. Make shure that when using the app at #2 an internal name is used like _appserver_ or _smtp_

**Sugested sollutions**

The bug(s) could be send upstream. At least it is odd that _/etc/resolv.conf_ is configured differently on different OSes. We could write a workaround by removing the `search .*` line.

**Comment by kvij** [comment]
Just confirmed that this issue also applies to Linux

**Comment by kvij** [comment]
Is this just a time constraint issue or truly never fix.
If it is let me know so I can investigate a workaround.

**Comment by kvij** [comment]
Seems to be a [regression](https://github.com/docker/for-win/issues/2301) that has been fixed again in docker 2.1.0.0. Not a lando issue
